packages/shared/ReactTypes.js

```diff
@@ -45,3 +45,7 @@ export type ReactPortal = {
   // TODO: figure out the API for cross-renderer implementation.
   implementation: any,
 };
+
+export type RefObject = {|
+  value: any,
```

Comment:
With input will be `ref.value.value` :)

Reply:
Yep, I believe `contents` is also a property on some DOM nodes too. `value` makes more sense in the case of this immutable object having a mutable object representing the ref itself.

Reply:
Might be a bit much, but what if it was `element` for a DOM element and `component` for a React component, instead of `value`?